servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -37,61 +37,101 @@ tracing                tracing-istio-system.127.0.0.1.nip.io
 
 +
 . Launch a browser and navigate to https://${KIALI_URL} (in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`).  You should see the Kiali console login screen.
-+
-image:kiali-login.png[Login Page]
-+
-Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
+
+. Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
 
 [[kiali-overview-page]]
 == Overview page
 
-After you log in you see the Overview page, which provides you with a quick overview of the health of the various namespaces in your system.
+When you first log in you see the Overview page, which provides a quick overview of the health of the various namespaces that are part of your {ProductShortName}.  You can choose to view the health of your applications, workloads, or services.
```

Comment:
Extra space after {ProductShortName}.

Reply:
I will never be able to unlearn two spaces after a period. it's baked into my muscle memory. I'm not that worried about it.